lms/djangoapps/courseware/access.py

```diff
@@ -848,8 +851,14 @@ def _has_catalog_visibility(course, visibility_type):
 def _is_descriptor_mobile_available(descriptor):
     """
     Returns if descriptor is available on mobile.
+
+    If the ignore_mobile_available_flag is enabled, the mobile_available flag
+    is ignored and access will be granted.
     """
-    return ACCESS_GRANTED if descriptor.mobile_available else MobileAvailabilityError()
+    if descriptor.mobile_available or IgnoreMobileAvailableFlagConfig.ignore_mobile_available_flag():
```

Comment:
Nitty: For readability, I would have checked for the flag before checking for the field - since the flag indicates whether or not to check for the field.  This is a minor point so feel free to ignore (or accept)  :).